CSC322-Grinnell/mayflower-directory
# Optimize directory

Resolves #143 

**We may find that this fails when we deploy it, but does fine in testing because it uses concurrency which could introduce bugs in any number of places.**

# Changes

- Fetch image URLs from AWS concurrently on directory page to improve performance.

# Testing

1. Start the server
2. Make sure the directory page loads
3. If images loaded before, make sure they still load
